scripts/gtcadis.py

```diff
@@ -148,7 +141,8 @@ def step1(cfg):
                    0.7523, 1.0041, 1.5083, 1.9958, 2.4657, 2.9082, 3.7269, 4.4834, 7.4896,
                    12.0153, 15.9873, 19.9191, 23.76])
     # Convert to Sv/s per photon FLUX (not fluence)
-    df = np.array([x * 1E-12 for x in df])
+    flux_magnitude = 1E12
```

Comment:
This changed from `1e-12` to `1e12` in the most recent commit - which is correct?

Related: what is the purpose of this anyway?